Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN go mod download
 COPY ./ ./
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o canipull main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o canipull main.go
 
 FROM ubuntu
```

Comment:
ubuntu is too big.

Reply:
try this one? mcr.microsoft.com/aks/devinfra/base-os-runtime-static:master.220630.1

Reply:
changed now